doc/index.rst

```diff
@@ -1,69 +1,94 @@
+#########################
+Electrometer
+#########################
+
+.. image:: https://img.shields.io/badge/SAL-API-gray.svg
+    :target: https://ts-xml.lsst.io/sal_interfaces/Electrometer.html
+.. image:: https://img.shields.io/badge/GitHub-gray.svg
+    :target: https://github.com/lsst-ts/ts_electrometer2
+.. image:: https://img.shields.io/badge/Jira-gray.svg
+    :target: https://jira.lsstcorp.org/issues/?jql=labels+%3D+ts_electrometer2
```

Comment:
ts_electrometer
again, consider grepping for all instances. there are probably a bunch.